src/styles/actor/_trade-dialog.scss

```diff
@@ -0,0 +1,9 @@
+#trade-dialog {
+    max-height: 67vh;
+    &:not(.maximizing, .minimizing, .minimized) {
+        min-height: 400px;
+    }
+    .window-content {
+        flex-direction: row;
+    }
+}
```

Comment:
This is an idea I had recently, but what's the viability of doing `global(#trade-dialog)` from the svelte application itself? Would that keeps things tidier?